@commitlint/rules/src/index.test.js

```diff
@@ -2,7 +2,7 @@ import path from 'path';
 import test from 'ava';
 import globby from 'globby';
 import {values} from 'lodash';
```

Comment:
note: `lodash` recommends to use:
```suggestion
import values from 'lodash/values';
```